docs/pages/organization.rst

```diff
@@ -0,0 +1,5 @@
+Repository / package organization
+====================================
+The below schematic image shows the organization of the repository and the packages.
+
+![Repo organization](repo_organization.png)
```

Comment:
I thought this figure was for our internal notes/organization, not really something to show to the public. The gist of it is good, but I think we can create a better/clearer image.

Reply:
Ah, good point! I'll think about how to mock something up.

Reply:
We can probably merge-as-is and then revisit this figure.